common.blocks/dropdown/dropdown.js

```diff
@@ -86,7 +86,10 @@ provide(bemDom.declBlock(this.name, /** @lends dropdown.prototype */{
         this.setMod('opened', data.modVal);
     }
 }, /** @lends dropdown */{
-    lazyInit : true
+    lazyInit : true,
+    onInit : function() {
+        this._events(Switcher).on('click', this.prototype.onSwitcherClick);
```

Comment:
in base declaration there is no `this.prototype.onSwitcherClick` — maybe we need default stub with `throw` about mandatory use some value of `_switcher` modifier